### DISCUSSION TITLE
My commit shows up as a commit by account "pyvirus"

### DISCUSSION THREAD

**Discussion by kittybwained** [community/community#48804] [Repositories] [answered]
### Select Topic Area

Bug

### Body

So for some reason [my repository](https://github.com/atlas-devel/WeepingOwlLinuxClient) has the "first commit" show up as a commit by the account "pyvirus". I have no idea what that account is, i do not own it. It also seems to have a badge with 0 contributions and was also made in 2013 with no repositories or any activity. I made this commit and push from the command line using the classic personal access token for authentication, under a pretty fresh installation of manjaro linux. I have no idea how this happened, but the code seems to be the exact same. Should i be worried that i'm infected, or is this a bug on github's end?

**Comment by airtower-luna** [comment, marked as answer]
The [email address in that commit](https://github.com/atlas-devel/WeepingOwlLinuxClient/commit/d28754efd1c8a4724fc5fe3580b9e07adfab3477.patch) (check the "From" line) looks odd at least, is that one that's on your account? If not, you probably need to fix the Git `user.email` setting on your computer, see [Setting your commit email address](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-email-preferences/setting-your-commit-email-address). The `user.email` is what Git records as the author email in commits you make, and GitHub uses that to assign commits to accounts.

**Reply by kittybwained** [reply]
i was using a fake email, the pyvirus account was probably using the same one
